docs/Home.md

```diff
@@ -79,12 +79,12 @@ the file was created, you will upload it to Jenkins in a subsequent step:
 1. Create a GKE cluster*:
     ```bash
     export CLUSTER=my-jenkins-cluster
-    export ZONE=us-central1-c
-    gcloud container clusters create --zone $ZONE $CLUSTER
+    export LOCATION=us-central1-c
+    gcloud container clusters create --zone $LOCATION $CLUSTER
```

Comment:
This command only has` --zone` or `--region` as available flags so this might be overkill.

Reply:
Yeah, I think we should keep the variable names consistent with the params.

Reply:
Done.